docs/bundles/SyliusResourceBundle/show_resource.rst

```diff
@@ -5,61 +5,62 @@ Your newly created controller service supports basic CRUD operations and is conf
 
 The simplest action is **showAction**. It is used to display a single resource. To use it, the only thing you need to do is register a proper route.
 
-Let's assume that you have a ``app.user`` resource registered. To display a single user, define the following routing:
+Let's assume that you have a ``app.book`` resource registered. To display a single Book, define the following routing:
 
 .. code-block:: yaml
 
-    # routing.yml
+    # app/config/routing.yml
 
-    app_user_show:
-        path: /users/{id}
+    app_book_show:
+        path: /books/{id}
         methods: [GET]
         defaults:
-            _controller: app.controller.user:showAction
+            _controller: app.controller.book:showAction
 
-Done! Now when you go to ``/users/3``, ResourceController will use the repository (``app.repository.user``) to find user with given id.
-If the requested user resource does not exist, it will throw a 404 exception.
+Done! Now when you go to ``/books/3``, ResourceController will use the repository (``app.repository.book``) to find a Book with the given id (``3``).
+If the requested user resource does not exist, it will throw a `404 Not Found` exception.
 
-When a user is found, the default template will be rendered - ``App:User:show.html.twig`` (like you configured it in `config.yml`) with the User result as the ``user`` variable.
-That's the most basic usage of the simple ``showAction`` action.
+When a Book is found, the default template will be rendered - ``App:Book:show.html.twig`` (like you configured it in the ``config.yml``)
+with the Book result as the ``book`` variable. That's the most basic usage of the simple ``showAction``.
 
 Using a Custom Template
 -----------------------
 
-Okay, but what if you want now to display same User resource, but with a different representation?
+Okay, but what if you want to display the same Book resource, but with a different representation in a view?
 
 .. code-block:: yaml
 
     # routing.yml
 
-    app_backend_user_show:
-        path: /backend/users/{id}
+    app_admin_book_show:
+        path: /admin/books/{id}
         methods: [GET]
         defaults:
-            _controller: app.controller.user:showAction
+            _controller: app.controller.book:showAction
             _sylius:
-                template: App:Backend/User:show.html.twig
+                template: App:Admin/Book:show.html.twig
```

Comment:
It is pretty obvious for us, where the file should be placed, but I'm afraid it could not be as simple for the community. Maybe we should present a folder structure or path at least?